README.md

```diff
@@ -27,17 +27,21 @@ Credits:
 
 ### Getting started and learning more
 
-The [latest version of HNC is
-v0.8.0](https://github.com/kubernetes-sigs/multi-tenancy/releases/tag/hnc-v0.8.0).
 To install HNC on your cluster, and the `kubectl-hns` plugin on your
-workstation, follow the instructions on that page. **NB: the release is in a
-different repo since we are currently migrating to this new repo. HNC v0.9.0
-will be released from this repo.**
+workstation, follow the instructions on our release pages.
+* [Latest version (v0.8.0)](https://github.com/kubernetes-sigs/multi-tenancy/releases/tag/hnc-v0.8.0)
+  * Note that this release is in a different repo since we are currently
```

Comment:
Worth stating anything about any potential patches to the old versions?

Reply:
I'm ok to deal with this on a case-by-case basis. We didn't have any patches for 0.6 or 0.7; the only reason we had so many patches for 0.5 was because we were holding 0.6 for the new APIs.

Reply:
Yeah I agree, was more refering to a statement saying we won't be patching old versions at all. Agreed though we can cross that bridge if it comes.